apps/studio/pages/api/projects/[ref]/run-lints.ts

```diff
@@ -2,6 +2,9 @@ import { NextApiRequest, NextApiResponse } from 'next'
 
 import apiWrapper from 'lib/api/apiWrapper'
 import { extractResponse } from 'pages/api/constants'
+import { constructHeaders } from 'lib/api/apiHelpers'
+import { post } from 'lib/common/fetch'
```

Comment:
can we swap these from the updated fetchers?

https://share.cleanshot.com/lZF17fL4

Reply:
Hi @saltcod , based on [this discussion](https://supabase.slack.com/archives/C03S3SW9485/p1718894630949059?thread_ts=1718893216.484629&cid=C03S3SW9485), we can keep using `post` from `lib/common/fetch`.